docs/cimpl6ObjectiveFHIR.md

```diff
@@ -2,7 +2,7 @@
 
 >**Note:** This documentation is in draft form.
 
-_This is an introductory guide to CIMPL's class library: **Objective FHIR Version 1.0**. The Guide assumes some knowledge of CIMPL. If you're looking for more information on CIMPL, see the [CIMPL 6 Tutorial](cimpl6Tutorial_detail.md), the [CIMPL 6 User Guide](cimpl6UserGuide.md), or the [CIMPL 6 Reference Manual](cimpl6Reference.md)._
+_This is an introductory guide to CIMPL's class library: **Objective FHIR Version 1.0**. The Guide assumes some knowledge of CIMPL. If you're looking for more information on CIMPL, see the [CIMPL 6 Tutorial](cimpl6Tutorial_detail.md), the [CIMPL 6 User Guide](cimpl6UserGuide.md), or the [CIMPL 6 Language Reference Manual](cimpl6LanguageReference.md)._
```

Comment:
Change from [CIMPL 6 User Guide] to [CIMPL Authoring] since it's the new title of the markdown page.

Reply:
OK, and carried this through in other locations. I suggest (and have) renamed the cimpl6UserGuide file to cimplAuthoring for consistency and to minimize confusion